docs/Rest-API.md

```diff
@@ -0,0 +1,147 @@
+# Axe REST API
```

Comment:
Overall, I think this document needs a lot more context. For example, a rest endpoint between what and what? What is the overall purpose of the spec? How about examples of what "arbitrary value" could be? What is the collection of results intended to cover: a scan of an entire app, a single element, multiple scans of individual elements,an entire app, multiple apps? How does a certain design make things easier for developers?

Reply:
If there's Context you feel your are lacking, given the fact that this doesn't exist yet, a quick call would be great! :)